test/utils.jl

```diff
@@ -29,5 +29,16 @@ end
     @test repeat(df, inner = 2, outer = 3) == ref
     @test repeat(view(df, 1:2, :), inner = 2, outer = 3) == ref
 end
+#
+# flatten(df::AbstractDataFrame, veccol::Union{Integer, Symbol})
+#
+@testset "flatten" begin
```

Comment:
can you please add tests having different containers in the flattened column (in particular things like: tuples, scalars, ranges, and mixtures of them will probably be typical use cases).